Index: EXPT12/p6animal.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>class Animal:\n    def sound():\n        print(\"Animals make sound\")\n\nclass Dog(Animal):\n    def sound():\n        print(\"Dogs bark\")\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/EXPT12/p6animal.py b/EXPT12/p6animal.py
--- a/EXPT12/p6animal.py	(revision 93e281dd0db5bc741f2a790e7108e71cc0ff1ddf)
+++ b/EXPT12/p6animal.py	(date 1743950062598)
@@ -4,5 +4,4 @@
 
 class Dog(Animal):
     def sound():
-        print("Dogs bark")
-
+        print("Dogs bark")
\ No newline at end of file
Index: EXPT10/p5palindrome.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/EXPT10/p5palindrome.py b/EXPT10/p5palindrome.py
new file mode 100644
--- /dev/null	(date 1743656445762)
+++ b/EXPT10/p5palindrome.py	(date 1743656445762)
@@ -0,0 +1,9 @@
+def palindrome(x):
+    x = x.lower()
+    if x == x[::-1]:
+        print("Given String is Palindrome")
+    else:
+        print("NOT A Palindrome")
+
+n = input("Enter a String: ")
+palindrome(n)
Index: EXPT10/p4even.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/EXPT10/p4even.py b/EXPT10/p4even.py
new file mode 100644
--- /dev/null	(date 1743655618390)
+++ b/EXPT10/p4even.py	(date 1743655618390)
@@ -0,0 +1,8 @@
+l = []
+n = int(input("Enter the no. of elements: "))
+for i in range(n):
+    ele = int(input("Enter Element:"))
+    l.append(ele)
+
+even = filter(lambda x: x % 2 == 0, l)
+print(list(even))
Index: EXPT12/p4circle.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/EXPT12/p4circle.py b/EXPT12/p4circle.py
new file mode 100644
--- /dev/null	(date 1743659134328)
+++ b/EXPT12/p4circle.py	(date 1743659134328)
@@ -0,0 +1,15 @@
+class Circle:
+    def __init__(self, r):
+        self.radius = r
+
+    def area(self):
+        ar = 3.14*(self.radius**2)
+        print(f"Area of the Circle is: {ar}")
+
+    def circumference(self):
+        cir = 2*3.14*(self.radius)
+        print(f"Circumference of the Circle is: {cir}")
+
+circle = Circle(7)
+Circle.area(circle)
+Circle.circumference(circle)
Index: EXPT12/p5student.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/EXPT12/p5student.py b/EXPT12/p5student.py
new file mode 100644
--- /dev/null	(date 1743688458450)
+++ b/EXPT12/p5student.py	(date 1743688458450)
@@ -0,0 +1,16 @@
+class Person:
+    fname = ""
+    lname = ""
+    def __int__(self,first, last):
+        self.fname = first
+        self.lname = last
+
+    def greet(self):
+        return f"Hello, my name is {self.fname} {self.lname}"
+
+class Student(Person):
+    sID = ""
+    def __init__(self, fname, lname, stuID):
+        super().__init__(fname,lname)
+        self.sID = stuID
+
