Index: EXPT10/p5palindrome.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/EXPT10/p5palindrome.py b/EXPT10/p5palindrome.py
new file mode 100644
--- /dev/null	(date 1743950357407)
+++ b/EXPT10/p5palindrome.py	(date 1743950357407)
@@ -0,0 +1,9 @@
+def palindrome(x):
+    x = x.lower()
+    if x == x[::-1]:
+        print("Given String is Palindrome")
+    else:
+        print("NOT A Palindrome")
+
+n = input("Enter a String: ")
+palindrome(n)
Index: EXPT10/p4even.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/EXPT10/p4even.py b/EXPT10/p4even.py
new file mode 100644
--- /dev/null	(date 1743950357404)
+++ b/EXPT10/p4even.py	(date 1743950357404)
@@ -0,0 +1,8 @@
+l = []
+n = int(input("Enter the no. of elements: "))
+for i in range(n):
+    ele = int(input("Enter Element:"))
+    l.append(ele)
+
+even = filter(lambda x: x % 2 == 0, l)
+print(list(even))
